src/dbus/client/thread_api_dbus.hpp

```diff
@@ -350,7 +350,162 @@ class ThreadApiDBus
      * @returns Error value otherwise
      *
      */
-    OtbrClientError GetSupportedChannelMask(uint32_t &aChannelMask);
+    ClientError GetSupportedChannelMask(uint32_t &aChannelMask);
+
+    /**
+     * This method gets the Thread routing locator
+     *
+     * @param[out]  aRloc16   The routing locator
+     *
+     * @retval OT_ERROR_NONE successfully performed the dbus function call
+     * @returns Error value otherwise
```

Comment:
Does this multi-line documentation provide value over something simple like:
`@returns A ClientError value.`
?

Reply:
Updated the doc string.
The error value behavior is returning ERROR_DBUS on dbus encoding/decoding errors. Otherwise we will return the return value of OpenThread api calls.